services/slack/intermediate.go

```diff
@@ -474,8 +488,25 @@ func (t *Transformer) TransformPosts(slackExport *SlackExport, attachmentsDir st
 
 			// bot message
 			case post.IsBotMessage():
-				// log.Println("Slack Import: bot messages are not yet supported")
-				break
+				if post.BotId == "" {
+					t.Logger.Warn("Unable to import the message as bot joining/leaving message is not supported")
```

Comment:
are u sure in this joining.leaving is the only case here

Reply:
updated the log